meson.build

```
@@ -25,6 +25,11 @@ version_name = 'v-'+version_major + '.' + version_minor
 cc = meson.get_compiler('c')
 host_os = host_machine.system()
 
+if host_os == 'windows' and cc.has_header_symbol('stdio.h', '_MSC_VER') and get_option('nuget-download')
+  run_command ('powershell.exe', 'Invoke-WebRequest https://dist.nuget.org/win-x86-commandline/latest/nuget.exe -OutFile $env:MESON_BUILD_ROOT\\nuget.exe')
```

Always split arguments (see https://mesonbuild.com/External-commands.html), even if they are key-values:

```suggestion
  run_command('powershell.exe',
              'Invoke-WebRequest', 'https://dist.nuget.org/win-x86-commandline/latest/nuget.exe',
              '-OutFile', '$env:MESON_BUILD_ROOT\\nuget.exe')
```